src/app/modules/server-settings/pages/manage-settings/manage-settings.component.ts

```diff
@@ -127,6 +127,23 @@ export class ManageSettingsComponent implements OnInit, OnDestroy {
         this.destroy.complete();
     }
 
+    get nonProxyHost(): FormArray {
+        return this.settingsForm.get('proxyConfigurationForm.nonProxyHosts') as FormArray;
+    }
```

Comment:
We're following the more generic convention - `getMethodName` and `setMethodName` instead of `get methodName` and `set methodName()`. Developers from any language would immediately spot and understand the pattern and it's more readable down the line.

```suggestion
    getNonProxyHosts(): FormArray {
        return this.getProxyConfigurationGroup('nonProxyHosts') as FormArray;
    }
```

Reply:
Also, the methods `getNonProxyHost` ,`addNonProxyHost` and `removeNonProxyHost` should be added after the `getProxyConfigurationGroup` for easier readability.

Reply:
This method is because I need to get the `nonProxyHosts` array and it is nested in a formGroup I can do this or create a variable to resolve this

Reply:
I changed to manage with a variable instead of this weird method